packages/website/tests/index.spec.ts

```diff
@@ -1,7 +1,30 @@
 import AxeBuilder from '@axe-core/playwright';
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 
-test('Index', async ({ page }) => {
-  await page.goto('/');
-  await new AxeBuilder({ page }).analyze();
+test.describe('Website', () => {
+  test('Axe', async ({ page }) => {
+    await page.goto('/');
+    await new AxeBuilder({ page }).analyze();
+  });
+
+  test('should have no errors', async ({ page }) => {
+    const errorMessages = [] as string[];
+    page.on('console', msg =>
+      msg.type() === 'error'
+        ? errorMessages.push(`[${msg.type()}] ${msg.text()}`)
+        : null,
+    );
+    await page.goto('/');
+    expect(errorMessages.length).toBe(0);
+  });
+  test('should have errors', async ({ page }) => {
+    const errorMessages = [] as string[];
+    page.on('console', msg =>
+      msg.type() === 'error'
+        ? errorMessages.push(`[${msg.type()}] ${msg.text()}`)
+        : null,
+    );
+    await page.evaluate(() => console.error('error'));
+    expect(errorMessages.length).toBe(1);
+  });
```

Comment:
This might be useful for development, but not necessary for actually testing the site.

```suggestion
```

(I'll end up making a library and/or finding one after this, I hope)